### ISSUE TITLE
In-Window App Switch + Grouping Apps✨ 

### ISSUE THREAD

**Issue by iltimasd** [SvelteLab/SvelteLab#144]
**Is your feature request related to a problem? Please describe.**
My biggest wish for the OG REPL was a way to categorize and easily switch between different REPLs, as my main use of the REPL is sandbox and break apart complex features or bugs. Allowing grouping allows me to quickly grok related REPLs and having the switcher in the window allows me to avoid context switching

**Describe the solution you'd like**
An additional pane to show other REPLs grouped by categories, Figma mockup:

![image](https://user-images.githubusercontent.com/20071081/230992529-79257cef-6937-4aec-a61d-e69536ba6da7.png)

**Describe alternatives you've considered**
_A clear and concise description of any alternative solutions or features you've considered._

No alternatives for grouping, need to remember part of the title, or manually search. No alternatives for switching app in REPL view, except maybe multiple tabs (however I use this and OG REPL as PWAs!!)

**Additional context**
_Add any other context or screenshots about the feature request here._
The user story around organizing might include other things, like tabs, a place to manage categories, etc

**Comment by SarcevicAntonio** [comment]
Something like this would be really useful for me too. Either for sharing cookbooks, or library examples, or as you said grouping bug explorations and fixes together. Good point about not having tabs in installed apps too!

Not sure if we maybe want tags instead of folders. Folders would probably enough for my needs, and tags feels a bit overkill?

**Comment by SarcevicAntonio** [comment]
Okay, so I just had some thoughts on this:

It could work like YouTube playlists, where one account can have multiple playlists, each with multiple videos inside, while one video can be in multiple playlists. They can be ordered, have a title and descriptions.

Videos could be called experiments and playlists could be called studies, to keep the SvelteLab science theme going.

There should be an overview page with shareable link for studies. There could also be a UI inside an opened experiment from a study which allows to quickly switch from one experiment to another (similar to the mockup).

The profile page could also be expanded to allow for customizable sections, similar to YouTube pages, but that's for after we ship the initial feature 👀

Basically this thought came because I want a collection of all the solved learn.svelte.dev SvelteKit challenges, similar to the examples page of Svelte. 

@iltimasd @paoloricciuti what do you think?

**Comment by K-Sato1995** [comment]
IMO, although it could complicate the UI a bit, would be nice to be able to switch Apps around.

**Comment by paoloricciuti** [comment]
> Okay, so I just had some thoughts on this:
> 
> It could work like YouTube playlists, where one account can have multiple playlists, each with multiple videos inside, while one video can be in multiple playlists. They can be ordered, have a title and descriptions.

This is cool, the reason i was going with a single category was because we had very little space to select the category. But if we can find a way to have multiple i think this is very beneficial.

> Videos could be called experiments and playlists could be called studies, to keep the SvelteLab science theme going.

I like the lab theme but i feel like experiments is way to long. We need something shorter and catchier imho. I'll ask Chat GPT for some ideas and report back.

> There should be an overview page with shareable link for studies. There could also be a UI inside an opened experiment from a study which allows to quickly switch from one experiment to another (similar to the mockup).

I was thinking that this might just be a command with an action_component. This will save space in the UI while also providing a way to always access your "playlists"

> The profile page could also be expanded to allow for customizable sections, similar to YouTube pages, but that's for after we ship the initial feature 👀

Yeah this is an easy win imho.

> Basically this thought came because I want a collection of all the solved learn.svelte.dev SvelteKit challenges, similar to the examples page of Svelte.

That's a supercool idea. More on that: in the future i want 100% implement something more in the style of codepen where you can actually search for repls and see recommended repls etc. I feel like the social aspect is another thing that we can enhance from the OG repl.

**Comment by SarcevicAntonio** [comment]
I think we should just call them Apps and Collections to be easily understandable.
